cloudberry/zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/impl/QueryPlanner.scala

```diff
@@ -302,25 +307,24 @@ object QueryPlanner {
 
     // No avg fields need to be merged, return immediately
     if (mergedField.isEmpty) {
-      result
+      return result
     }
-    else {
-      //merge sum and count to avg
-      val handledRows = result.value.map(row => {
```

Comment:
I didn't change any of the following code, just get rid of the `else` wrap.
* Previous:
```
if (something empty) {
    result
}
else {
    do something
    result
}
```
* Now:
```
if (something empty) {
    return result
}

do something
result (This is syntax of scala, the last sentence is the returned result by default, no explicit return statement needed)
```